LimeBean/BeanApi.cs

```diff
@@ -573,6 +573,10 @@ public partial class BeanApi : IDisposable, IBeanCrud, IBeanFinder, IDatabaseAcc
             set { Storage.ConvertEmptyStringToNull = value; }
         }
 
+        /// <summary>
+        /// Gets or Sets whether integers are detected and extracted if in string type variables, when storing to the database.
+        /// Query paramaterisation and the database still handle storage types
+        /// </summary>
         public bool RecognizeIntegers {
```

Comment:
Okay thanks for the insight!
I've created issue #11 just as a research point, to see if we can save ourselves some useless conversions, but am committed to not making breaking changes where the behaviour has an effect on the outcome of Store.